src/Raven.Client/Util/JavascriptConversionExtensions.cs

```diff
@@ -1113,7 +1120,7 @@ public override void ConvertToJavascript(JavascriptConversionContext context)
                         {
                             writer.Write(parameter.Name.Substring(2));
                             writer.Write(".");
-                            name = name.Replace("<>h__TransparentIdentifier", "h__TransparentIdentifier");
+                            name = name.Replace(TransparentIdentifier, "h__TransparentIdentifier");
```

Comment:
Better to put `"h__TransparentIdentifier"` also as a const next to `internal const string TransparentIdentifier`

Reply:
done